services/web/server/tests/unit/with_dbs/03/wallets/test_wallets.py

```diff
@@ -209,3 +212,24 @@ async def test_wallets_events_auto_add_default_wallet_on_user_confirmation(
     assert wallet.description == _WALLET_DESCRIPTION_TEMPLATE.format(user_name)
     assert mock_rut_sum_total_available_credits_in_the_wallet.called
     assert mock_add_credits_to_wallet.called == product.is_payment_enabled
+
+    # Test whether default wallet was set in user preferences
+    url = client.app.router["get_default_wallet"].url_for()
+    resp = await client.get(f"{url}")
+    data, _ = await assert_status(resp, web.HTTPOk)
+    assert data
+    wallet = WalletGetWithAvailableCredits(**data)
+    assert wallet.available_credits > Decimal(0)
+
+
+@pytest.mark.parametrize("user_role,expected", [(UserRole.USER, web.HTTPOk)])
```

Comment:
have at least one test per route that test access-right. For example see 
https://github.com/ITISFoundation/osparc-simcore/blob/master/services/web/server/tests/unit/with_dbs/03/login/test_login_unregister.py#L20-L41